src/parser.coffee

```diff
@@ -301,7 +301,7 @@ class VASTParser
             companionAd.width = companionResource.getAttribute("width")
             companionAd.height = companionResource.getAttribute("height")
             for htmlElement in @childsByName(companionResource, "HTMLResource")
-                companionAd.type = htmlElement.getAttribute("creativeType") or 0
+                companionAd.type = htmlElement.getAttribute("creativeType") or 'text/html'
```

Comment:
From the looks of it, when declaring a `StaticResource` it is common place to apply a `creativeType` to the field... however, this does not seem to be the case with `HTMLResource` and `IFrameResource`

BrightCove: http://support.brightcove.com/en/video-cloud/docs/supported-vast-20-elements
```
[SUPPORTED]    <StaticResource creativeType="" />
[SUPPORTED]    <IFrameResource/>
[SUPPORTED]    <HTMLResource/>
```
VAST Spec: http://www.iab.net/media/file/VAST-2_0-FINAL.pdf
CompanionAds > HTMLResource and IFrameResource have no attributes